CONTRIBUTING.md

```diff
@@ -238,7 +238,7 @@ If you have commit access, please follow this process for merging patches and cu
 8. If the commit messages follow [our conventions](@commit-message-conventions)
```

Comment:
If someone of the core members have a link to a document with conventions, i would update the link too.

Reply:
Thanks for noticing @big-r81 and for starting the discussion https://github.com/apache/couchdb/discussions/3918

I suggested to use some minimal rules here. Wonder what others think

Reply:
Looks like we might have a proposal for commit message guidelines in the discussion. @big-r81 if you get a chance, mind updating the PR with it so we can see how it looks? Up to you if it would look better as a paragraph or a section somewhere.

Thank you again for contributing and for staring the discussion around this issue. It's much appreciated!

Reply:
Hi @nickva , I added your hints from the discussion. Please take a look.